.github/workflows/ci.yml

```diff
@@ -9,10 +9,7 @@ jobs:
     name: Lint and Test with Node.js ${{ matrix.node }}
     strategy:
       matrix:
-        node:
-          - 14
-          - 16
-          - 18
+        node: [18, 20]
```

Comment:
Please preserve the formatting style

Reply:
Sure. However, it appears that some dependencies are not compatible with node.js 20. Is yarn default to set the `engines-strict` config flag?